src/chapters/strings/encoding-characters.rst

```diff
@@ -0,0 +1,215 @@
+Encoding Characters
+===================
+
+.. index:: ! bit, character encoding
+
+If you had microscope powerful enough to view the data stored on a computer's
```

Comment:
had a* microscope

Reply:
We'll need to update the JS book on that one, too.